packages/bind.rb

```diff
@@ -3,21 +3,13 @@
 class Bind < Package
   description 'BIND is open source software that enables you to publish your Domain Name System (DNS) information on the Internet, and to resolve DNS queries for your users.'
   homepage 'https://www.isc.org/downloads/bind/'
-  version '9.10.5-p3'
-  source_url 'https://www.isc.org/downloads/file/9-10-5-p3/?version=tar-gz'
-  source_sha256 '8d7e96b5b0bbac7b900d4c4bbb82e0956b4e509433c5fa392bb72a929b96606a'
+  version '9.12.0'
+  source_url 'https://www.isc.org/downloads/file/bind-9-12-0/'
+  source_sha256 '29870e9bf9dcc31ead3793ca754a7b0236a0785a7a9dc0f859a0bc42e19b3c82'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/bind-9.10.5-p3-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/bind-9.10.5-p3-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/bind-9.10.5-p3-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/bind-9.10.5-p3-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '5d0ffc0fdfadccb9f838730797e645c249444abb077e5b3d05ad8f0a8241924f',
-     armv7l: '5d0ffc0fdfadccb9f838730797e645c249444abb077e5b3d05ad8f0a8241924f',
-       i686: '8d52384957c720b527650357ad3e5fd3732fbcae2083613c68993b913f603855',
-     x86_64: '3ca89eb998a259f393920174ac2bd1427465101eeb2a9cc8dcb5a880bbe0e4e0',
   })
 
   depends_on "buildessential"
```

Comment:
This line can be removed since it's now part of Chromebrew "core".

Reply:
Please abstract the install location like below:
```
- system "BUILD_CC=gcc ./configure --with-randomdev=no --with-ecdsa=yes --with-gost=yes --prefix=/usr/local"
+ system "BUILD_CC=gcc ./configure \
                --with-randomdev=no \
                --with-ecdsa=yes \
                --with-gost=yes \
                --prefix=#{CREW_PREFIX} \
                --libdir=#{CREW_LIB_PREFIX}"
```

Reply:
To which line # are you referring? re: This line can be removed since it's now part of Chromebrew "core".

Reply:
Both suggestions done.